Code-4-Community/speak-for-the-trees-frontend
# Improve Tree Page-Map Return

## Why

[ClickUp Ticket](https://app.clickup.com/t/1dk42tb)


Users are having difficulty switching between the map and tree pages. Improves user flow by opening the tree page in a new tab and opening the map at the street zoom level at the site location via the 'Return to Tree Map' button.

## This PR



- use Link target to open tree page in a new tab
- use location state to pass the map the site's location and the zoom level to go to
  - ensure layer visibility is initialized according to the zoom level rather than assuming the map is zoomed out
- clean up codebase by putting the redirect button under themed components

## Verification Steps

 
Tested going from the map to the tree page (check that page opens in a new tab) and returning to the map (check that the map is zoomed in at the site's location)
Tested going from a tree page to the map via the return button (check that the map is zoomed in at the site's location)
